keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -115,7 +115,7 @@ data class ClusterSpec(
     health: HealthSpec?,
     scaling: Scaling?,
     tags: Map<String, String>?,
-    overrides: Map<String, ServerGroupSpec>
+    overrides: Map<String, ServerGroupSpec> = emptyMap()
```

Comment:
One thing I noticed when creating cluster definitions is I needed to include `overrides: {}` in the YAML. I suspect overrides will only rarely need to be specified so not requiring users to specify it at all if they don't need to is the right thing to do.

Reply:
yes this is definitely better